src/main/scala/uk/gov/hmrc/ct/ct600e/v2/E1030.scala

```diff
@@ -20,5 +20,6 @@ import uk.gov.hmrc.ct.box._
 import uk.gov.hmrc.ct.ct600e.v2.retriever.CT600EBoxRetriever
 
 case class E1030(value: Option[String]) extends CtBoxIdentifier("Claimer's name") with CtOptionalString with Input with ValidatableBox[CT600EBoxRetriever]{
-  override def validate(boxRetriever: CT600EBoxRetriever): Set[CtValidation] = validateStringAsMandatory("E1030", this)
+  override def validate(boxRetriever: CT600EBoxRetriever): Set[CtValidation] =
+    validateStringAsMandatory("E1030", this) ++ validateOptionalStringByLength("E1030", this, 2, 56) ++ validateOptionalStringByRegex("E1030", this, validNonForeignLessRestrictiveCharacters)
```

Comment:
Well - I haven't created it and that's the name it's being referred to in all of the documents about XMLs and validation.